src/inpaint.jl

```diff
@@ -15,8 +15,9 @@ abstract type InpaintAlgo end
 Inpaint `img` on pixels marked as `true` in `mask` using
 algorithm `algo`.
 """
-function inpaint(img::AbstractArray{T,N}, mask::AbstractArray{Bool,N},
-                 algo::InpaintAlgo) where {N,T}
+function inpaint(img::AbstractArray{<:AbstractGray}, mask::AbstractArray{Bool,N},
+                algo::InpaintAlgo) where {N}
```

Comment:
Same here, there is a misalignment in line 19. The `a` should appear below `i`. Let's be picky with these details because otherwise the code becomes difficult to read.

Reply:
Also, for both function signatures, we could perhaps drop the type annotations, leaving only the type of the `algo`. Putting type annotations in `img` and `mask` is nowadays considered a Julia antipattern. We are restricting the generality of the algorithm prematurely when we do so.